rain_api_core/general_util.py

```diff
@@ -1,7 +1,29 @@
 import logging
 import os
 import sys
-
+import json
+import time
+
+def return_timing_object(**timing):
+    timing_object = { "service": "Unknown", "endpoint": "Unknown", "method": "GET", "duration": 0, "unit": "milliseconds"}
+    timing_object.update({k.lower(): v for k,v in timing.items()})
+    return {"timing":timing_object }
+    
+def d(time_in):
```

Comment:
I would prefer a less cryptic name for something that's used outside of whatever module it lives in.